### PR TITLE
docs: Document magma_deb VM

### LINKED ISSUE #13685: Run integration tests against debian installation in CI
Tasks:

- [x] Add a new GHA workflow that is triggered. It should be triggered on master after the build-all step is done. Plan: Copy the existing integ test job and modify it.
- [x] Investigate the stability issues. You can reproduce them by running the test suite on your machine and waiting until pipelined crashes. @nstng may know more details.
- [x] Optional: Look into how the debian packages are chosen. If we run after a merge to master, try to make sure it's using the results of the merge. In the current state, the process would always take the latest version of artifacts uploaded to the artifactory. Options:
  - Is there a way to get the version tag from the previous build-all run?
  - Another option: https://github.com/marketplace/actions/repository-dispatch
  - Add the commit hash to the uploaded version (this might cause other issues).
- [x] Add documentation to docusaurus.
- [x] Prepare internal review meeting.

Following on from #13263, we should create a version of the LTE integration test workflow that uses the debian installation as opposed to the make installation. Note that there are still some issues with the debian VM to be ironed out, see #13551.  Once working, the debian VM should be documented in the docusaurus with reference to running the integration tests.

* Add a new CI workflow to run the tests. It will not be required yet, and it will be an additional job, not replace the existing LTE integration tests. Run the workflow after build-all.
* Investigate stability issues and pipelined crashes. Try to improve stability.
* Optional: Look into how the debian packages are chosen. If we run after a merge to master, try to make sure it's using the results of the merge.

### PR DESCRIPTION
**Pull request by voisey**
## Summary

Document how to run the integration tests versus the magma_deb VM.

## Test Plan

`cd $MAGMA_ROOT/docs && make precommit` passes locally.

### CODE REVIEW

File: docs/readmes/lte/s1ap_tests.md [hunk 1]
```diff
@@ -19,23 +19,41 @@ interfere with the test scenario.
...
 acts as the gateway between *magma_test* and *magma_trfserver*.
 
 ### Gateway VM setup
 
+There are two options for setting up the gateway VM, with the difference being the magma installation
+method: it can either be installed via `make` or from a debian package, the latter being the
+method by which magma is usually deployed. For everyday development, the `make` installation is
+recommended, while the debian installation is useful for testing packages before release.
+
+> **Warning**: These two VMs use the same network configuration, so only one of them should be
```
Comment (Neudrino) [inline docs/readmes/lte/s1ap_tests.md]: `should` seems a bit too nonrestrictive here? Should this not be `must not`?

File: docs/readmes/lte/s1ap_tests.md [hunk 2]
```diff
@@ -19,23 +19,41 @@ interfere with the test scenario.
...
+#### Make installation
+
 Spin up and provision the gateway VM, then make and start its services:
 
 1. From `magma/lte/gateway` on the host machine: `vagrant up magma && vagrant ssh magma`
 1. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
 
+#### Debian installation
+
+Spin up the *magma_deb* VM. The services should start automatically:
```
Comment (Neudrino) [inline docs/readmes/lte/s1ap_tests.md]: We might want to remove the `should` here? That would sound more confident.

**Review by Neudrino — APPROVED** [review]
Formally sound.

File: docs/readmes/lte/s1ap_tests.md (hunk 1 above)
Comment (voisey) [reply docs/readmes/lte/s1ap_tests.md]: I changed "should" to "must" and reworded it slightly. Is it better now?

File: docs/readmes/lte/s1ap_tests.md (hunk 2 above)
Comment (voisey) [reply docs/readmes/lte/s1ap_tests.md]: Done :slightly_smiling_face:

**Review by ajahl — APPROVED** [review]

File: docs/readmes/lte/s1ap_tests.md [hunk 3]
```diff
@@ -19,23 +19,41 @@ interfere with the test scenario.
...
+#### Make installation
+
 Spin up and provision the gateway VM, then make and start its services:
 
 1. From `magma/lte/gateway` on the host machine: `vagrant up magma && vagrant ssh magma`
 1. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
 
+#### Debian installation
+
+Spin up the *magma_deb* VM. The services start automatically:
```
Comment (nstng) [inline docs/readmes/lte/s1ap_tests.md:52]: ~"During provisioning the latest magma gateway debian build from the magma artifactory is installed. This is, the deployed gateway might not match your local repository state."

**Review by nstng — APPROVED** [review]
lgtm. one suggestion for additional information - feel free to ignore it, might be enough that this is documented in code.

File: docs/readmes/lte/s1ap_tests.md (hunk 3 above)
Comment (voisey) [reply docs/readmes/lte/s1ap_tests.md:52]: Nice suggestion, thanks! I've added this as a warning.

**Pull request merged**